CHANGELOG.md

```diff
@@ -1,3 +1,29 @@
+## v2.1.0
+Wed  1 Apr 2020 16:12:36 BST
+
+* [1917562](https://github.com/hyperledger/fabric-chaincode-node/commit/1917562) [FABCN-392](https://jira.hyperledger.org/browse/FABCN-392) Update to Node 12
```

Comment:
Previously we've hand edited this file to remove some of the 'minutae'... eg, Re-enable tests... etc.. Just to be the major functional changes.